node/src/blockchain/blockchain_interface.rs

```diff
@@ -200,38 +195,55 @@ impl BlockchainInterface for BlockchainInterfaceClandestine {
         _new_fingerprints_recipient: &Recipient<PendingPayableFingerprintSeeds>,
         _accounts: &[PayableAccount],
     ) -> Result<Vec<ProcessedPayableFallible>, PayableTransactionError> {
-        error!(self.logger, "Can't send transactions out clandestinely yet",);
+        error!(
+            self.logger,
+            "Trying to send transactions with uninitialized blockchain interface. \
+        The blockchain service URL was not set."
+        );
         Err(PayableTransactionError::Sending {
-            msg: "invalid attempt to send txs clandestinely".to_string(),
+            msg: "invalid attempt to send transactions".to_string(),
             hashes: vec![],
         })
     }
 
     fn get_gas_balance(&self, _address: &Wallet) -> ResultForBalance {
-        error!(self.logger, "Can't get eth balance clandestinely yet",);
-        Ok(0.into())
+        error!(
+            self.logger,
+            "Trying to get eth balance with uninitialized blockchain interface. \
+        The blockchain service URL was not set."
+        );
+        Ok(U256::zero())
```

Comment:
Philosophically, it is an error to try to get the consuming-wallet balance without a blockchain service URL. If we return zeros from these two methods, will we fool other code into doing stupid things because it thinks there's a real wallet with a real zero balance? Or are those "stupid" things actually not so stupid? Do they turn out to be exactly the things that should be done when you're consuming services without a blockchain service URL?